src/app/pages/LiquidityMining/components/RemoveLiquidityDialog/RemoveLiquidityDialogV1.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useEffect, useMemo, useState } from 'react';
 import { useTranslation, Trans } from 'react-i18next';
+import type { AbiItem } from 'web3-utils';
```

Comment:
```suggestion
import { AbiItem } from 'web3-utils';
```

Reply:
Why not to use `import type`? It increases readability so it clears that import is used as type only and not value. Also, to my understanding, prevents for value from being included to bundle after build (as JS doesn't need to know about types anymore and class/interface is not actually used anywhere)

Reply:
You can probably leave it but it just seemed weird to me because we don't use `import type` anywhere in the app. I didn't know about this feature before but I read the documentation and it seems handy sometimes (although I don't think we need this kind of optimization in our case).

Anyway, you can leave as it is.